code/src/java/plugin/exporttokens/SpellMemToken.java

```diff
@@ -258,7 +258,9 @@ else if ("DURATION".equals(aLabel))
 						}
 						else if ("DESC".equals(aLabel) || "EFFECT".equals(aLabel))
 						{
-							String mString = aPC.parseSpellString(selectedCSpell, aPC.getDescription(aSpell));
+							String mString = aPC.getInfoToken(".INFO.DESC", aSpell);
+							if (mString.equals(".INFO.DESC"))
```

Comment:
@mertonmonk 
CheckstyleMain doesn't allow IFs without curly brackets, because the code might be misleading. Should append be within IF, or not?
What variant you meant?
<img width="718" alt="image" src="https://github.com/user-attachments/assets/fd945ae1-c8d8-4ed7-86d0-5fd5e3ebb4b7" />
or
<img width="728" alt="image" src="https://github.com/user-attachments/assets/327233b0-8e95-40da-8db7-9f6d9ff1f087" />
thanks

Reply:
The append should be outside of the if statement so it appends mString whether it was re-defined within the if statement or not.

Reply:
Thanks for pointing out the gradlew target.  It's been quite a while since I did much coding for PCGen, so there is a lot of new code and I'm not particularly familiar with gradle.  I wish I had the time and energy to pull out all the interfaces - I think code should be simple so it's easier for newer developers to understand.